README.md

```diff
@@ -19,6 +19,7 @@ that returns the instance of `MessageDef` implementation for the current message
 ## Requirements
 - Java 11
 - Gradle `6.9.x`
```

Comment:
In this section, let's also explain why it is so right now, and that in the future releases we are planning to support more recent Gradle versions, and so is Spine.